src/hooks/accounts/graphql/GetAccounts.query.graphql

```diff
@@ -1,17 +1,15 @@
-
 query GetAccounts {
-    lastBlock @client {
-        parachainBlockNumber,
-        relaychainBlockNumber
-    }
-    
-    accounts @client {
-        id,
-        name,
-        isActive,
-        balances {
-            assetId,
-            balance
-        }
+  lastBlock @client {
```

Comment:
This is fine, it will refetch balances with every new block as long as this query is active.

Reply:
would be c00l if we used fragments for the lastBlock fields, or even the whole lastBlock piece itself.